qcfractal/interface/collections/torsiondrive_dataset.py

```diff
@@ -32,13 +32,22 @@ class TorsionDriveSpecification(BaseModel):
 class TorsionDriveDataset(BaseProcedureDataset):
     class DataModel(BaseProcedureDataset.DataModel):
 
-        records: Dict[str, TDRecord] = {}
+        records: Dict[str, TDEntry] = {}
         history: Set[str] = set()
-        specs: Dict[str, TorsionDriveSpecification] = {}
+        specs: Dict[str, TDEntrySpecification] = {}
 
         class Config(BaseProcedureDataset.DataModel.Config):
             pass
 
+    def _internal_compute_add(self, spec: Any, entry: Any, tag: str, priority: str) -> ObjectId:
```

Comment:
So I have noticed that on this and the `GOEntrySpecification` the `_internal_compute_add` only adds them as services, which I agree is the correct way to compute them, but couldn't they have also been added as large single compute tasks in the past? Do we want to remove that ability?

Reply:
Only TD can be run as a procedure and we have yet to try that. In general, I don't think running TD as a single procedure is a good idea as it creates a chance for a failure at the very end consuming *a lot* of CPU time.